build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -321,6 +324,9 @@ fun configureTests() {
                         enabled && !rerunAllTests
                     }
                 )
+                if (testEnabled && enabled.get()) {
```

Comment:
This adds the custom value when the test is configured, not when it is executed. Would it make sense to add the custom value in a `doFirst`, so it would only happen when the task actually runs? Or should this happen by inspecting if the test task is in the task graph?

We also need to make sure this value is configuration cache compatible. I think it currently is, though I am not sure.

Like I said before, I think it should be fine to capture `enabled && !rerunAllTests` only on the root project, given that we set that via project properties, and then add a tag. I think this would be simpler right now. The build scan will show anyway which tests did run with PTS.

Reply:
Let's do the simple solution now.